index.js

```diff
@@ -3,10 +3,24 @@ const app = express();
 require('dotenv').config();
 const cors = require('cors');
 const PORT = process.env.PORT
+const mongoose = require('mongoose');
+const PASTE_DB = process.env.PASTE_DB
+const pasteBinRoute = require('./route/pasteBinRoute');
+
+
+const connectDB = async () =>{
+    await mongoose.connect(PASTE_DB, {useNewUrlParser: true , useUnifiedTopology: true})
+    console.log('MongoDB');
+}
+connectDB();
 
 // bodyparser
 app.use(express.json());
-app.use(express.urlencoded({extended: true}))
+app.use(express.urlencoded({extended: true}));
+
+// Route
+
+app.use(pasteBinRoute);
 
 // cors
 app.use(cors({origin:"*" , credentials: true , methods:  ['GET','POST','DELETE','UPDATE','PUT','PATCH']}))
```

Comment:
Why all this config (especially the methods bit)?

Reply:
I used this method app.use(cors()) , the frontend developers were still complaining about cors  so i read up on cors and i came up with this method and it actually works

Reply:
I see it works, I'm just wondering why it's this verbose

Reply:
I will take out the methods